code/modules/petro/petro_reactions.dm

```diff
@@ -0,0 +1,108 @@
+/datum/chemical_reaction/fermi/cruderefine
+	name = "Crude oil refining"
+	id = "crudefine"
+	results = list("asphalt" = 0.3, "fueloil" = 0.3, "diesel" = 0.2, "kerosene" = 0.1, "naphtha" = 0.05, "butane" = 0.05)
+	required_reagents = list("crudeoil" = 5, "water" = "1" )
+	mix_message = "the mixture splits into fractions."
+	required_temp = 1
+	required_container = /obj/machinery/plumbing/reaction_chamber
+	//FermiChem vars:
+	OptimalTempMin 		= 900 		// Lower area of bell curve for determining heat based rate reactions
+	OptimalTempMax 		= 2300 		// Upper end for above
+	ExplodeTemp 		= 2800 		// Temperature at which reaction explodes
+	OptimalpHMin 		= 5.5 		// Lowest value of pH determining pH a 1 value for pH based rate reactions (Plateu phase)
+	OptimalpHMax 		= 7 		// Higest value for above
+	ReactpHLim 			= 0.1 		// How far out pH wil react, giving impurity place (Exponential phase)
+	CatalystFact 		= 0 		// How much the catalyst affects the reaction (0 = no catalyst)
+	CurveSharpT 		= 8 		// How sharp the temperature exponential curve is (to the power of value)
+	CurveSharppH 		= 9 		// How sharp the pH exponential curve is (to the power of value)
+	ThermicConstant		= -10 		// Temperature change per 1u produced
+	HIonRelease 		= -0.02 		// pH change per 1u reaction (inverse for some reason)
+	RateUpLim 			= 1 		// Optimal/max rate possible if all conditions are perfect
+	FermiChem 			= TRUE		// If the chemical uses the Fermichem reaction mechanics
```

Comment:
These numbers seem a little high. 
To confirm, you want a yield of 0.7u reactant to 6u product? (almost a 10x increase?)

Additionally, for each 0.7u of reactant consumed, you'll be cooling the reaction by 50K. Your CurveSharpT is also 8(!!) which means your polynomial is to the 8th power, which is nuts. Basically, until 2000K your yield will tiny, even slower than synthtissue, which takes all round to make.
![image](https://user-images.githubusercontent.com/33956696/69008966-79900b80-0948-11ea-99b7-7efec5622db0.png)
This is your reaction kinetics, considering 1000K is usually the upper limit chemists can reach, it seems unreactable atm, especially with such an endothermic reaction, or rather, way too slow for it to be a viable power method.

There are equal problems when we consider the pH of the system, your window uses a 9th order polynomial, effectively making it a shear drop, so if you ever step out of the window, you'll tank your pH pretty fast, and depending on if it's low volume, it'll likely explode just as fast.
![image](https://user-images.githubusercontent.com/33956696/69009103-3e8ed780-094a-11ea-8515-fa0b967f924d.png)

I assume you got your pH window from https://ph.endressdigital.com/en-in/oil-gas/ ? This is hydro cracking, and their limits are more based on the machine they used rather than the kinetics of the system itself, so I would encourage you to reconsider your window. Equally, -0.02 might not seem like much of a displacement, it can quickly add up, and I assume that pH is difficult to adjust currently? If we look at [This](https://pdf.sciencedirectassets.com/277910/1-s2.0-S1876610212X00055/1-s2.0-S1876610212001932/main.pdf?X-Amz-Security-Token=IQoJb3JpZ2luX2VjEGcaCXVzLWVhc3QtMSJHMEUCIASmVGwvywvnitc1wYlFHfbltm0A5FEcpNvg3X1TkJCnAiEAj5SF%2FynfY%2BveUCYDciOJFyRPzjGve%2B%2FVZ5YgUeche%2BEq2QIIj%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FARACGgwwNTkwMDM1NDY4NjUiDEG6QVC%2BNEmhb7BweiqtAqgyP7YqXJ%2FxtLVxIoz8ez29iCUqOwC879hfzsNhZhSEBca0nTFBGhsYpa8QosRtMTE5X6GvGsggHrTfzk0onrFZEnBqPruLOYSb8WCIufDE0rfZofP8caEuT2tdxJOfoeqkLlNAF0GLwPDiCkRcOW80U2LwBVWwCg8pD9OhNMGBdVaQYjvQW%2Bn%2BjYkBRnx8fPXG6F1mDBbmcM0mBlTCJ4Mq0yGW9B5zcKGY%2BGHxQ092YCFUnJ3Sgjl1m8DV4gWpfFDm%2FgQ3L84WXhyyH4pkrnHmnGWQqQidlu6BGggmddLN%2BYg5uMeCDU%2Blcznpd6ybA9thMEBz%2B9IvYIlcnRHu3dOQYfxp4ecPbICBmn1UzsPKpJlxoUzIPEal7r%2B3wwV98VsTg89ToWuVp86ZGeUwq6fF7gU6zwLlhE%2FXYtmOfQAFVE3ur0rVpQacsbu93A7kF0vNw7J3h6k%2FzxKk2bAscI%2FtwPEH8HCsnhMM4NaAs3%2FUgzAfPo6Y%2FGj54r%2F0StCT1OZ94PpBvwwXQtX816WhGMuYYBLlBt25ffFUjxAsD0R5tUCJAPFPwpUPkOukqJR3It8R9g9lmv3QzG3ebQubhAx4DR8V41bjyzj5KV4LhJMhsPCnB9ZczE7BWG0QBN7l4cujeluPM4k27Rzahir%2FyZjZW7k0YYrgdRjPxeP9Ryh7C9HqvrpSQ1leHfw8yvTnew6D%2BkLge5XUK9NfVsWgV2L3Mahm4RxUrZJZ7AMPZMxdRcGfEVeOXmwNZfUb8b2FWMhR80OngK7AhZ8pQxOZZqvS%2FW4xceYC1YyXnprwmPziV73mC61KFZWtRejosqNBHD511VGAWzMtggZ33dnkrYgHWX%2FtGA%3D%3D&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20191117T150431Z&X-Amz-SignedHeaders=host&X-Amz-Expires=300&X-Amz-Credential=ASIAQ3PHCVTYYOZA5LXF%2F20191117%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=64949a652bd9770bd9604c16ea794f4f61a0718379e6dd3ae004d5d10c116fca&hash=d696574935fd0e5dcb29a064d347cf4f8608231efc34d3ba5caf75065d9bffbc&host=68042c943591013ac2b2430a89b270f6af2c76d8dfd086a07176afe7c76c2c61&pii=S1876610212001932&tid=spdf-c12767da-96d8-4f8a-a661-4d0312070e41&sid=a860c9548f449547623812a8b77a09e3ec0cgxrqb&type=client) we can see that a pH of 5-10 is viable for crude oil, where pH only affects the viscosity of the oil, which, unless we code fluid dynamics, shouldn't matter for our simplified system.

As a side note, if we're trying to match reality, cracking isn't really a reaction, it's more a distillation, which is not a process that follows the Arrhenius equation, hence you wouldn't model it in quite the same way. I realise we don't have distillation methods atm, so we have to make do with what we have. Also, because I can't help myself, zeolite is usually used as a catalyst; it's used for the cracking (the saturation) of unsaturated components to turn unvaluable hydrocarbons into more valuable ones, which could be a side reaction using fermimechanics, while keeping ditry conversion a viable and easy option. (Especially since I plan to dump this onto engineers to power a whole "station" with in a future map). 

Though, reality I don't think should be strived for in this case, because it'll either be way too intense, divisive and confusing In my opinion. It also feels like you based these off the current fermichems, which uses extremes of the system to balance the power of the product (I.e. it's intensive, consuming, and you often end up with a smaller ammount). I think that quick and dirty methods should be easily producible, then we can build more expansive/involving methods that produce better power. At the very least you should edit your pH ranges to be wider, your inoptimal ranges to be much wider, say 2-12, otherwise purity is a nonfactor. Additionally your CurveSharps are way too big, I'd say drop them down by a whole order of magnitude, reduce your thermic constant to -2, and HIon a little, depending on ease of pH adjustment, or natural mechanics based upon testing.